### ISSUE TITLE
Some UI love to error pages

### ISSUE THREAD

**Issue by peterblazejewicz** [aspnet/templating#292]
Can the error pages be redesigned?
At least can the white space be added to clearly separate the content (important information to the developer) from the viewport edge? (the readability is being hurt IMO)

Thanks!

![20180202224338](https://user-images.githubusercontent.com/14539/35756273-eee0c78e-086a-11e8-8566-4e21ae02f017.jpg)
![20180202223845](https://user-images.githubusercontent.com/14539/35756274-ef001b2a-086a-11e8-9423-9303d293e9da.jpg)

**Comment by Eilon** [comment]
Can you point out exactly where the issue is? It's not clear to us from the images where the problem is. Thanks!

**Comment by peterblazejewicz** [comment]
Yes, the error is delivered, but we could do better. At the moment it starts on the very edge of the viewport.
I think people don't print (or kindle) books with text spanning from the very edge of the page, the same we do on the web:
https://www.smashingmagazine.com/2009/09/10-useful-usability-findings-and-guidelines/

To be more academic:
http://www.humanfactors.com/newsletters/yeah_but_can_you_give_me_a_reference.asp
https://www.sciencedirect.com/science/article/pii/S074756320300075X

Thanks!

**Comment by peterblazejewicz** [comment]
And that mostly valid for:
- SPA templates - there is no page layout which adds the white space, so when the error hit in production mode the content is rendered without white space
- MVC templates, when there is an problem with rendering template for some reason - usually in MVC templates the error page is used with layout, which has white space, but on some (one I've done a screenshot) it renders incorrectly

Maybe the same page (error page) can be used here - that does not depend on layout engine (at least in SPA templates) and just render an error page
thanks!
